container/test-app/wc/clientAPI.html

```diff
@@ -84,6 +84,7 @@ <h3>
 
       [...document.querySelectorAll('luigi-container')].forEach(luigiContainer => {
         luigiContainer.addEventListener(MFEventID.NAVIGATION_REQUEST, event => {
+          // alert("NAVIGATION_REQUEST received");
```

Comment:
This alert might be removed - same applies to line 115

Reply:
oops missed these, thanks 👍